src/main/java/frc/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.subsystems.shooter;
+
+public class ShooterConstants {
+
+  private static final String CONSTRUCTOR_EXCEPTION = "constant class";
+
+  private ShooterConstants() {
+    throw new IllegalStateException(CONSTRUCTOR_EXCEPTION);
+  }
+
+  public static final boolean DEBUGGING = false;
+  public static final boolean TESTING = true;
+  public static final String SUBSYSTEM_NAME = "Shooter";
+
+  public static final int TOP_SHOOTER_MOTOR_ID = 27;
+  public static final int BOTTOM_SHOOTER_MOTOR_ID = 28;
+  public static final int ANGLE_MOTOR_ID = 29;
+  public static final int ANGLE_ENCODER_ID = 9;
+
+  public static final int SETPOINTCOUNT = 5;
```

Comment:
make sure that you are fixing all warnings (e.g., this should be SET_POINT_COUNT)